tests/source/issue-3779/lib.rs

```diff
@@ -1,3 +1,4 @@
+// rustfmt-unstable: true
```

Comment:
File `tests/config/issue-3779.toml`:

```toml
unstable_features = true
ignore = [
  "tests/**/issue-3779/ice.rs"
]
```

The included file already contains the `unstable` flag, but I think it's not picked by the following logic:

https://github.com/rust-lang/rustfmt/blob/d35ebf72e50a6b11491f9392c80fd392fed365cd/src/test/mod.rs#L493-L494

I think it's fine to just remove the flag from the included config.

Reply:
So remove `unstable_features = true` from `tests/config/issue-3779.toml`?